index.html

```diff
@@ -8,129 +8,34 @@
-                <li><a href="https://github.com/benjp009" target="_blank" rel="noopener" class="fa fa-github"></a></li>
-              </ul>
-            </div>
-          </div>
+  <!-- Header Section  -->
+    <section class="header">
+      <!--- Top navigation Bar --->
+      <div class="topnav">
+        <a href="#default" class="logo">My Logo</a>
+        <div class="nav-menu" id="myLinks">
+          <a href="#portfolio">Portfolio</a>
+          <a href="#about">About</a>
+          <a href="#contact">Contact</a>
         </div>
+        <!-- "Hamburger menu" / "Bar icon" to toggle the navigation links -->
+        <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+          <i class="fa fa-bars"></i>
+        </a>
       </div>
 
-      <!-- The "works" categories are here -->
-      <section class="grid-container">
-        <section class="grid-item">
-            <img class="work-section-image" src="ressources/Work-1.png" alt="Picture for the first work of Benjamin Patin">
-            <h2>Tonic</h2>
-            <ul class="client-info">
-              <li class="client-name">CANOPY</li>
-              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
-              <li class="project-style">Back End Dev</li>
-              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
-              <li class="project-year">2015</li>
-            </ul>
-            <p class="project-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <ul class="language-used">
-              <li><div class="tags">html</div></li>
-              <li><div class="tags">css</div></li>
-              <li><div class="tags">javaScript</div></li>
-            </ul>
-          <button type="button" class="button">See Project!</button>
-        </section>
-        <section class="grid-item">
-            <img class="work-section-image" src="ressources/Work-2.png" alt="Picture for the first work of Benjamin Patin">
-            <h2>Multi-Post Stories</h2>
-            <ul class="client-info">
-                <li class="client-name">CANOPY</li>
-                <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
-                <li class="project-style">Back End Dev</li>
-                <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
-                <li class="project-year">2015</li>
-              </ul>
-            <p class="project-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <ul class="language-used">
-              <li><div class="tags">html</div></li>
-              <li><div class="tags">css</div></li>
-              <li><div class="tags">javaScript</div></li>
-            </ul>
-          <button type="button" class="button">See Project!</button>
```

Comment:
- [ ] [**Optional**] It is very nice to alternate alignment of images on the project section as given in the figma design, but it would be nice to also match text alignment as seen in the design.

#### Yours (text right aligned)
![image](https://user-images.githubusercontent.com/61377322/139525302-23c1bcb9-79c2-44c9-80d9-3cbeb4b04825.png)
#### Design (text left aligned)
![image](https://user-images.githubusercontent.com/61377322/139525358-c254bfcb-fe93-4a4b-9516-43629cf69d21.png)